src/typography/types.ts

```diff
@@ -0,0 +1,16 @@
+export type TypographyWeight = "regular" | "semibold" | "bold" | "light";
+
+export interface TypographyProps extends React.HTMLAttributes<HTMLElement> {
+    //Weight prop such as regular, bold
+    weight?: TypographyWeight | undefined;
```

Comment:
comments should be in jsdoc format `/** <description here> */` so that they are visible in vscode